### DISCUSSION TITLE
Custom url actions

### DISCUSSION THREAD

**Discussion by omniraptorr** [pdelfan/ouranos#7] [Ideas]
The default bluesky client has a button to translate a post, but it is hardcoded to google translate. I was thinking that this url based query could be configurable and support multiple services. So in the settings you could specify a custom url string and insert various variables such as post did, op's handle, etc. a bit like custom search engines in a firefox.

Possible use cases are alternative translation services (google translate kinda lags behind the competition), third party apps that provide different views (like skythread), or archive services like archive.ph

I could mb mockup some UI but hopefully you get the idea

**Comment by pdelfan** [comment]
Sorry I missed this. I really like this idea, what other use cases do you see for this? In terms of UI, I imagine it could be represented as an "Actions/extensions" dropdown where it'll have all the actions that users have added (links to other translators, archive.ph, open post in another app, etc.)
